AdminUi/src/AdminUi/AdminUi.csproj

```diff
@@ -15,6 +15,9 @@
     <ItemGroup>
         <PackageReference Include="Microsoft.AspNetCore.SpaProxy" Version="7.0.8" />
         <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="8.0.0" />
+        <PackageReference Include="OpenIddict" Version="4.3.0" />
+        <PackageReference Include="OpenIddict.EntityFrameworkCore" Version="4.3.0" />
+        <PackageReference Include="OpenIddict.Server" Version="4.3.0" />
```

Comment:
I forgot to mention: can you check which of these references are actually necessary? It's probably only the `OpenIddict.EntityFrameworkCore` one